packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -60,7 +60,7 @@
   "error.contentTypeName.reserved-name": "This name cannot be used in your project as it might break other functionalities",
   "error.validation.enum-duplicate": "Duplicate values are not allowed",
   "error.validation.enum-empty-string": "Empty strings are not allowed",
-  "error.validation.enum-number": "Values cannot start with a number",
+  "error.validation.enum-regex": "At least one value contains invalid characters. Values should always have an alphabetical character preceding any number.",
```

Comment:
Preceding the `first` number maybe?
Otherwise, it would mean that you couldn't have `a12` as `2` is preceded by a number and not an alphabetical character.
What do you think?

Reply:
Yeah, that makes sense 👍🏼

Reply:
Adressed in 1973fb57f6afd68de29492f1e21b850bfe0f8268